resourcemanager/commonschema/zone.go

```diff
@@ -55,3 +55,12 @@ func ZoneSingleComputed() *schema.Schema {
 		Computed: true,
 	}
 }
+
+// ZoneSingleOptionalComputed returns the schema used when a single Zones can be returned
```

Comment:
I think we for completeness we should update the comment, maybe to something like the below?

```suggestion
// ZoneSingleOptionalComputed returns the schema used when a single Zone can be specified or a single Zone is returned when omitted
```